File: backend/internal/avaliation/domain/score.go
package domain

type Score struct {
	DiagnosticoPrimario     string `json:"diagnostico_primario"`
	DiagnosticoSecundario   string `json:"diagnostico_secundario"`
	DomicilioRisco          bool   `json:"domicilio_risco"`
	ImpedimentoDeslocamento bool   `json:"impedimento_deslocamento"`
	AlimentacaoParenteral   string `json:"alimentacao_parenteral"`
	AspiracaoTraqueo        string `json:"aspiracao_traqueo"`
	VentilacaoMecanica      string `json:"ventilacao_mecanica"`
	MedicacaoParenteral     string `json:"medicacao_parenteral"`
	Atividades              bool   `json:"atividades"`
	Banho                   bool   `json:"banho"`
	Vestir                  bool   `json:"vestir"`
	HigienePessoal          bool   `json:"higiene_pessoal"`
	Transferencia           bool   `json:"transferencia"`
	Continencia             bool   `json:"continencia"`
	Alimentacao             bool   `json:"alimentacao"`
	EstadoNutricional       string `json:"estado_nutricional"`
	ViaAlimentacaoMedicacao string `json:"via_alimentacao_medicacao"`
	InternacoesUltimoAno    string `json:"internacoes_ultimo_ano"`
	AspiracaoViasAereas     string `json:"aspiracao_vias_aereas"`
	Lesoes                  string `json:"lesoes"`
	ViaMedicacoes           string `json:"via_medicacoes"`
	ExerciciosVentilatorios string `json:"exercicios_ventilatorios"`
	UsoOxigenoterapia       string `json:"uso_oxigenoterapia"`
	NivelConsciencia        string `json:"nivel_consciencia"`
}

File: web-forms/src/store/evaluation/score.js
import { defineStore } from 'pinia';

export const useScoreStore = defineStore('score', {
    state: () => ({
        score: {
     
            diagnostico_primario: '',
            diagnostico_secundario: '',
            domicilio_risco: false,
            impedimento_deslocamento: false,

            alimentacao_parenteral: null,
            aspiracao_traqueo: null,
            ventilacao_mecanica: null,
            medicacao_parenteral: null,


            banho: null,
            vestir: null,
            higiene_pessoal: null,
            transferencia: null,
            continencia: null,
            alimentacao: null,
            atividades: null,

    
            estado_nutricional: null,
            via_alimentacao_medicacao: null,
            internacoes_ultimo_ano: null,
            aspiracao_vias_aereas: null,
            lesoes: null,
            via_medicacoes: null,
            exercicios_ventilatorios: null,
            uso_oxigenoterapia: null,
            nivel_consciencia: null
        }
    }),
    actions: {
        reset() {
            this.score = {
                diagnostico_primario: '',
                diagnostico_secundario: '',
                domicilio_risco: false,
                impedimento_deslocamento: false,
                alimentacao_parenteral: null,
                aspiracao_traqueo: null,
                ventilacao_mecanica: null,
                medicacao_parenteral: null,
                banho: null,
                vestir: null,
                higiene_pessoal: null,
                transferencia: null,
                continencia: null,
                alimentacao: null,
                estado_nutricional: null,
                via_alimentacao_medicacao: null,
                internacoes_ultimo_ano: null,
                aspiracao_vias_aereas: null,
                lesoes: null,
                via_medicacoes: null,
                exercicios_ventilatorios: null,
                uso_oxigenoterapia: null,
                nivel_consciencia: null
            };
        }
    }
});